scripts/o3de/ExportScripts/export_source_android.py

```diff
@@ -0,0 +1,246 @@
+    has_errors = exp.process_command(gradle_build_command, cwd=target_android_project_path)
+    if has_errors:
+        raise exp.ExportProjectError("Gradle Build Failure.")
+
+    if deploy_to_device:
+        
+        activity_name = f'{ctx.project_name}Activity'
+        logger.info(f"Activity name set to {activity_name}")
+
+        
+        org_name = f'org.o3de.{ctx.project_name}'
+        logger.info(f"Organization name set to  {org_name}")
+
+        signing_config_file_path = target_android_project_path /'app/build.gradle'
+
+        try:
+            build_gradle_content = signing_config_file_path.read_text('utf-8','ignore')
+            match_signing_config = re.compile("(signingConfigs\\s).*(storeFile\\s).*(storePassword\\s).*(keyPassword\\s).*(keyAlias\\s).*", re.DOTALL)
+            signing_config_verified = match_signing_config.search(build_gradle_content) is not None
+            if signing_config_verified:
+                android_support.deloy_to_android_device(target_android_project_path,
+                                                        build_config,
+                                                        android_sdk_home,
+                                                        org_name,
+                                                        activity_name)
+            else:
+                logger.error("Unable to verify signing configuration in build.gradle. Aborting deployment...")
+        except FileNotFoundError:
+            logger.error("Unable to open build.gradle file. Aborting deployment...")
+        
+
+def export_source_android_parse_args(o3de_context: exp.O3DEScriptExportContext,
+                                     export_config: command_utils.O3DEConfig,
+                                     android_config: command_utils.O3DEConfig):
+    parser = argparse.ArgumentParser(
+                    prog=f'o3de.py export-project -es {__file__}',
+                    description="Exports a project as an Android APK that is optionally deployed to an android device. "
+                                "In order to use this script, the engine and project must be setup and registered beforehand. ",
+                    epilog=exp.CUSTOM_CMAKE_ARG_HELP_EPILOGUE,
+                    formatter_class=argparse.RawTextHelpFormatter,
+                    add_help=False
+    )
+    
+    eutil.create_common_export_params_and_config(parser, export_config)
+
+    parser.add_argument('--deploy-to-android',action='store_true',help='At completion of build, deploy to a connected android device.')
+ 
+    parser.add_argument('-B', '--build-dir', type=str, 
+                                          help=f"The location to write the android project scripts to. Default: '{android.DEFAULT_ANDROID_BUILD_FOLDER}'", 
+                                          default=android.DEFAULT_ANDROID_BUILD_FOLDER) 
```

Comment:
For consistency sake, lets follow the pattern used in the [export_source_ios_xcode.py](https://github.com/o3de/o3de/blob/11dc29fce65bd8037db9b3a4dc8254ec5ec2c101/scripts/o3de/ExportScripts/export_source_ios_xcode.py#L117-L118) and use '-abp', '--android-build-path' instead.